src/Digdir.Domain.Dialogporten.Infrastructure/Altinn/Authorization/DecisionRequestHelper.cs

```diff
@@ -297,6 +298,36 @@ private static XacmlJsonMultiRequests CreateMultiRequests(
     private static List<AltinnAction> SortForXacml(this List<AltinnAction> altinnActions) =>
         altinnActions.OrderBy(x => x.Name).ThenBy(x => x.AuthorizationAttribute).ToList();
 
+    internal static void XacmlRequestRemoveSensitiveInfo(object obj)
+    {
+        var type = obj.GetType();
+        var properties = type.GetProperties()
```

Comment:
Her vet vi vel typene, så vi slipper å gjøre `if (obj is XacmlJsonAttribute` etc. lenger nedi her. 
type.GetProperties() kan også caches unna så vi slipper å gjøre dette oppslaget hver gang det skjer (kun 1 gang ved oppstart)